README.md

```diff
@@ -57,6 +57,9 @@ Currently, the following Extended CSS pseudo classes are supported:
 - `:contains(text / regexp)`: [[ADG reference]](https://github.com/AdguardTeam/ExtendedCss#extended-css-contains)
 - `:has-text(text / regexp)`: [[uBO reference]](https://github.com/gorhill/uBlock/wiki/Procedural-cosmetic-filters#subjecthas-textneedle)
 - `:if-not(selector)`: [[ADG reference]](https://github.com/AdguardTeam/ExtendedCss#extended-css-if-not)
+- `:matches-css-after(arg)`: [[uBO reference]](https://github.com/gorhill/uBlock/wiki/Procedural-cosmetic-filters#subjectmatches-css-afterarg)

```

Comment:
Hm, we don't have it explained in the ExtCSS repo?

Reply:
@ameshkov Maybe they are deprecated, as i saw AdGuard Extended CSS converts `:matches-css-before` and `:matches-css-after` to `:matches-css`: https://github.com/AdguardTeam/ExtendedCss/blob/b1b3184a219bb04d3edc5efde8e502323e177a89/test/selector/converter.test.ts#L41-L59

Reply:
yeah, we try to avoid adding new pseudos when it can be achieved with existing ones.

we should still document that @slavaleleka 

does not hold this PR though

Reply:
The documentation currently says the following about these two pseudo classes:

>  Obsolete pseudo-classes `:matches-css-before()` and `:matches-css-after()` are supported for better compatibility.

Reply:
everything's good then:)